test/utils/selectors.js

```diff
@@ -0,0 +1,67 @@
+export function confirmCheckDetails(value) {
```

Comment:
Id probably reorganise this file into some selector constants, e.g. export const TERMS_AND_CONIDITONS_CHECKBOX = '#terms' 

and then group all the functions together, i.e. export function getTypeOfLivestockSelector (they should be named like that, I think).